### PR TITLE
Add files on how to run cell contamination removal module

### LINKED ISSUE #92: Add files on how to run cell contamination removal module
## What are the scientific goals of the analysis?

To add files on how to run the `cell-contamination-removal-analysis` module.

### PR DESCRIPTION
**Pull request by AntoniaChroni**
# Purpose/implementation Section

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## What scientific question/module of analysis is your analysis addressing?

To add files on how to run the `cell-contamination-removal-analysis` module.

## What GitHub issue does your pull request address?

Closes #92.

## Directions for reviewers 
Tell potential reviewers what kind of feedback you are soliciting.

### Which areas should receive a particularly close look?

Please review for documentation and code logic and clarity.

## Reproducibility Checklist

- [x] The dependencies required to run the code in this pull request have been added to the project Dockerfile.

## Documentation Checklist

- [x] This analysis module has a `README` and it is up to date.
- [x] The analytical code is documented and contains comments.

### CODE REVIEW

File: analyses/cell-contamination-removal-analysis/README.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# Pipeline for removing cell contamination, and repeating normalization and integration steps analysis for sc-/sn-RNA-Seq Analysis in 10X Genomics data
```
Comment (sharonfreshour) [inline analyses/cell-contamination-removal-analysis/README.md:1]:
```suggestion
# Pipeline for removing cell contamination, and repeating normalization and integration steps of analysis for sc-/sn-RNA-Seq Analysis in 10X Genomics data
```

File: analyses/cell-contamination-removal-analysis/run-cell-contamination-removal-analysis-steps-4.R [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+root_dir <- yaml$root_dir
+analysis_dir <- file.path(root_dir, "analyses", "cell-contamination-removal-analysis") 
+report_dir <- file.path(analysis_dir, "plots") 
+
+################################################################################################################
+# step 4 - Run markers
+future_globals_value = yaml$future_globals_value_clustering_module
+resolution = yaml$resolution_find_markers
+
+rmarkdown::render('02-find-markers.Rmd', clean = TRUE,
```
Comment (sharonfreshour) [inline analyses/cell-contamination-removal-analysis/run-cell-contamination-removal-analysis-steps-4.R:32]: I believe the `Rmd` script called here should be `04-find-markers.Rmd` to be consistent with file structure from README and numbering of `Rmd` scripts called in `steps-1-2-3`.

```suggestion
rmarkdown::render('04-find-markers.Rmd', clean = TRUE,
```

**Review by sharonfreshour — COMMENTED** [review]
It appears the `run-cell-contamination-removal-analysis-steps-4.R` script needs to be updated to run `04-find-markers.Rmd` instead of `02-find-markers.Rmd`.

I also suggested one change to the README to address a minor typo.

Otherwise, the code looks good, it is clear and seems logical.

File: analyses/cell-contamination-removal-analysis/README.md (hunk 1 above)
Comment (AntoniaChroni) [reply analyses/cell-contamination-removal-analysis/README.md]: Addressed here f6abf4efaa373911176e3ee9df7c1333f3df704f.

**Review by sharonfreshour — APPROVED** [review]

**Pull request merged**